pages/integrations/inputs/o365_input.rst

```diff
@@ -0,0 +1,54 @@
+.. _o365_input:
+
+***********************
+Office 365 Integrations
+***********************
+
+.. attention:: This is a Graylog Enterprise feature and is only available since Graylog version 3.3. A valid Graylog Enterprise license is required.
+               It is meant for the hosted "online" office and do not use this to collect logs from a windows server, in your data center.
+
+Prerequisites
+=============
+You have an application in Azure AD, a subscription to Office 365 and a subscription to Azure.
+
+Why does this plugin exist?
+============================
+Office 365 Integrations plugin, collects data from Windows Logs, Email, File Sharing and Data Loss Prevention Logs in one input.
+This plugin retrieves data for each tenant, using continuous polling and stores the data(user activity) in `elastic <https://en.wikipedia.org/wiki/Elasticsearch>`_ to detect threats.
+
+How do I get this plugin running?
+==================================
+you should be able to login into the following websites with your exempli@gratia.onmicrosoft.com account.
+
+ - `office 365 <https://www.office.com/?auth=2>`_
+ - `azure Portal <https://portal.azure.com/#home>`_
+ - `turn on <https://docs.microsoft.com/en-us/microsoft-365/compliance/turn-audit-log-search-on-or-off?view=o365-worldwide#turn-on-audit-log-search>`_ unified logging to see results.
+
+once you login into Azure portal,you can obtain the essentials for running this plugin.Please collect items 1-3 from the azure portal.
+Leave the defaults for the rest of the fields while navigating the **graylog-o365-input-plugin** wizard.You must be up and running with this input **Office 365 Log Events**
+plugin in a few minutes.If the plugin stops **RUNNING** completely, due to any failures, click **Start input** to collect data from the last **CHECKPOINT_TIME** of failure.
+
+
+1) ``Directory (tenant) ID``
+2) ``Application (client) ID``
+3) ``Client Secret``
+4) ``Enterprise and GCC Government plans`` is the most common subscription type.
+
+This `input <http://localhost:8080/system/inputs>`_ plugin polls data internally using, https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-reference.
+Click `Show metrics <http://localhost:8080/system/metrics/node/node-id?filter=filterid>`_, `Show Received Messages <http://localhost:8080/search?q=gl2_source_input%3A5f1b38dc2fb55336f12afc1a&rangetype=relative&relative=0>`_
+to validate if the data is flowing in .
```

Comment:
We could also put a link or description to clicking on the "Show Messages" from the input page on the O365 input to show them too